addons/stock/models/stock_warehouse.py

```diff
@@ -69,6 +70,8 @@ class Warehouse(models.Model):
         'stock.location.route', 'supplied_wh_id', 'Resupply Routes',
         help="Routes will be created for these resupply warehouses and you can select them on products and product categories")
     show_resupply = fields.Boolean(compute="_compute_show_resupply")
+    sequence = fields.Integer(default=1,
```

Comment:
Increase the default sequence number (e.g 10), the purpose is to avoid having the new warehouses in the top at the creation.